BlueprintUI/Tests/AttributedTextTests.swift

```diff
@@ -0,0 +1,129 @@
+import XCTest
+@testable import BlueprintUI
+
+class AttributedTextTests: XCTestCase {
```

Comment:
Nit: It would also be good to add some test using emojis and non-ascii characters to validate this does what we expect there as well.

Reply:
Done